tensorflow/contrib/rnn/python/kernel_tests/rnn_test.py

```diff
@@ -0,0 +1,215 @@
+"""Tests for rnn module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import time
+import timeit
+
+import numpy as np
+from six.moves import xrange  # pylint: disable=redefined-builtin
+
+import tensorflow as tf
+from tensorflow.python.kernel_tests import rnn_test
+
+tf.contrib.rnn.Load()
```

Comment:
I borrowed this code from:
tensorflow/contrib/tensor_forest/python/ops/training_ops.py

quote:


# Workaround for the fact that importing tensorflow imports contrib

# (even if a user isn't using this or any other contrib op), but

# there's not yet any guarantee that the shared object exists.

# In which case, "import tensorflow" will always crash, even for users that

# never use contrib.


--
William Chan
Carnegie Mellon University
(650) 450-9455
williamchan.ca

On Wed, May 11, 2016 at 9:50 PM, ebrevdo <notifications@github.com> wrote:

> In tensorflow/contrib/rnn/python/kernel_tests/rnn_test.py
> <https://github.com/tensorflow/tensorflow/pull/2002#discussion_r62964826>:
>
> > +"""Tests for rnn module."""
> > +
> > +from __future__ import absolute_import
> > +from __future__ import division
> > +from __future__ import print_function
> > +
> > +import time
> > +import timeit
> > +
> > +import numpy as np
> > +from six.moves import xrange  # pylint: disable=redefined-builtin
> > +
> > +import tensorflow as tf
> > +from tensorflow.python.kernel_tests import rnn_test
> > +
> > +tf.contrib.rnn.Load()
>
> Not sure if we discussed this... why can't we just Load() inside the
> *init* for tf.contrib.rnn?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2002/files/8b6f595036ec7829cb78f985883f3e579254dc69#r62964826>
>